models/sipnet/man/remove.config.SIPNET.Rd

```diff
@@ -1,14 +1,14 @@
+% Generated by roxygen2 (4.0.2): do not edit by hand
```

Comment:
These tags shouldn't be in individual Rd files anymore -- these are showing up because you're using an out-of-date version of roxygen -- you'll want to update and re-roxygenize